frontend/src/features/projects/list/ProjectListView.tsx

```diff
@@ -32,6 +35,9 @@ interface IProjectFilterState {
   assessWorkflow?: boolean;
 }
 
+const getPropertyReportUrl = (filter: IProjectFilter) =>
```

Comment:
This is a project report, not a property report.

Reply:
Additionally you'll need to create a way to download two types of Excel documents.  The first is the general project report, the second is the Surplus Property List report.

1) Generic Project List - GET `/api/reports/projects` or POST `/api/reports/projects/filter`
2) Maggie's Financial Report - GET `/api/reports/projects/surplus/properties/list` or POST `/api/reports/projects/surplus/properties/list/filter`

The first one is essentially identical to what we did for the Properties List View, whereas the second is to all Maggie to continue using her Excel spreadsheet for reporting.